server/routes.coffee

```diff
@@ -28,6 +28,10 @@ module.exports = (app, router) ->
     .get (req,res) ->
       calendarController.getCalendarEvents(req,res)
 
+  router.route   "/sendMeetingInvite"
+    .post bearer, (req,res) ->
+      meetingController.sendEmailInvites(req,res)
```

Comment:
Lets comment this out for the moment... just so it doesn't send random email invites when we are doing testing, then we can make a PR for re-enabling the sending email invites.